discovery/build.sbt

```diff
@@ -0,0 +1,3 @@
+libraryDependencies += "com.typesafe.akka" % "akka-http-experimental_2.11" % "2.4.11.2"
```

Comment:
we use 2.12, akka-http is not experimental, please update dependedncies
`"com.typesafe.akka" %% "akka-http" % "10.0.9"`